change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Doesn't appear to change the lib in any way
```suggestion
  "type": "none",
```

Reply:
yes, updated that. Thanks!